PraceDomowe/PracaDomowa6/Sobieski Bartlomiej/Sobieski_Bartlomiej_PD6.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<p>Bardzo dobrze, chociaż warto zauważyć, że ten wykres prezentuje trochę co innego -- na stronie inquirer pokazana jest skumulowana liczba dawek i to jest główna pokazana informacja, u Pana główną informacją jest liczba dawek danego producenta. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>